src/AppInstallerRepositoryCore/Microsoft/Schema/1_7/Interface.h

```diff
@@ -8,6 +8,10 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_7
 {
     using namespace std::string_view_literals;
 
+    // Version 1.7
+    static constexpr std::string_view s_MetadataValueName_MapDataFolded = "mapDataFolded"sv;
+    static constexpr char s_MetadataValue_MapDataFolded_Separator = ';';

```

Comment:
I guess we would want to remove the corresponding entries in MetadataTable.h?

Reply:
They were removed.